src/celutil/formatnum.cpp

```diff
@@ -192,26 +191,23 @@ bool getNumberSymbol(const UNumberFormat *numFormat, UNumberFormatSymbol symbol,
 #ifdef USE_ICU
 NumberFormatter::NumberFormatter()
 {
-    const char *orig = N_("LANGUAGE");
-    const char *lang = _(orig);
-
     UErrorCode status = U_ZERO_ERROR;
-    UNumberFormat* numFormat = unum_open(UNUM_DECIMAL, nullptr, 0, lang == orig ? "en" : lang, nullptr, &status);
+    UNumberFormat* numFormat = unum_open(UNUM_DECIMAL, nullptr, 0, nullptr, nullptr, &status);
```

Comment:
Idea was to have 
```c++
NumberFormatter::NumberFormatter(const char *locale)
{
 UErrorCode status = U_ZERO_ERROR;
UNumberFormat* numFormat = unum_open(UNUM_DECIMAL, locale != nullptr ? locale : "en", 0, nullptr, nullptr, &status);
...
```

Reply:
@ajtribick using ICU's default locale does have some a downside. The one that i notice is that on Windows, ICU's default locale does not respect the "LANG" environment variable while gettext respects it.

So probably still a better idea to use "LANGUAGE" from .po file? The only downside for I can think of using this is that we cannot adapt to some differences in locales like between en_GB and en_US for date formatting since en is passed in.

Reply:
The LANG environment variable isn't the usual way to do language changes on Windows, so that isn't necessarily fatal. If we need LANG support on Windows, then would probably make sense to do a call to [uloc_setDefault](https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/uloc_8h.html#aa86430509b310792443d05e5bda643b3) at startup if this environment variable is set.

Reply:
then I'm keeping this part of code as is @375gnu . it will be up to frontends to decide whether uloc_setDefault should be called or not.

Reply:
okay